examples/fc/main.tf

```diff
@@ -0,0 +1,103 @@
+# Specify the provider and access details
+provider "alicloud" {
+  account_id = "${var.account}"
+  region = "${var.region}"
+}
+
```

Comment:
I think this provider alicloud can be ignored. "AccountId" can be get automatically by AK.